cilium/cilium.io
# feat: develop events page

**This PR:**
Adds an Events page:
- Updated "Filters" and "Pagination" components for use in "Events" and "Blog";
- Adds links to the footer and menu "News and Media";
- Adds necessary events.

TODO before merge:
- [x]  Delete test events;

**Steps to test**

- Open the [Events page](https://deploy-preview-279--cilium.netlify.app/events/)
- Open the [Blog page](https://deploy-preview-279--cilium.netlify.app/blog/)
- Confirm that everything looks and works as expected
